certified-connectors/SignHost/apiDefinition.swagger.json

```diff
@@ -0,0 +1,945 @@
+          }
+        },
+        "Language": {
+          "description": "The language of the sender notifications and the receipt, only de-DE, en-US, es-ES, fr-FR, it-IT, pl-PL and nl-NL are allowed.",
+          "type": "string",
+          "enum": [
+            "de-DE",
+            "en-US",
+            "es-ES",
+            "fr-FR",
+            "it-IT",
+            "pl-PL",
+            "nl-NL"
+          ]
+        },
+        "Seal": {
+          "type": "boolean",
+          "default": false,
+          "description": "Seal the document before sending to the signers."
+        },
+        "Signers": {
+          "type": "array",
+          "description": "The signer information.",
+          "items": {
+            "type": "object",
+            "required": [
+              "Email"
+            ],
+            "properties": {
+              "Id": {
+                "type": "string",
+                "description": "The id of the signer, must be unique within a transaction.\nIf you don't provide an id we will generate one for you.\n"
+              },
+              "Email": {
+                "type": "string",
+                "description": "The e-mail address of the signer",
+                "example": "john.doe@example.com"
+              },
+              "IntroText": {
+                "type": "string",
+                "description": "An intro text to show to the user during the sign process.\nThis will be shown on the first screen to the signer and supports limited markdown markup.\nThe following markup is supported:\n- `# Headings`\n- `*Emphasis*` / `_Emphasis_`\n- `**Strong**` / `__Strong__`\n- `1. Ordered` and `- Unordered` lists\n"
+              },
+              "Authentications": {
+                "description": "List of authentications that the signer has to authenticate with.\nThe order in which the authentications are provided determine in which order the signer will have to perform the specified method.\nAuthentications must be performed before the document(s) can be viewed.\nYou **must** explicitly specify the API-version when using this feature.\nThis is done with the header: 'Accept: application/vnd.signhost.v1+json'.\n",
+                "type": "array",
+                "items": {
+                  "$ref": "#/definitions/Authentication"
+                }
+              },
+              "Verifications": {
+                "description": "List of verifications that the signer has to verify with.\nThe order in which the verifications are provided determine in which order the signer will have to perform the specified method.\nVerifications must be performed before the document(s) can be signed.\nYou **must** use one of the following verifications as the last method:\n- Consent\n- itsme sign<sup>*</sup>\n- PhoneNumber\n- Scribble\n- SigningCertificate<sup>*</sup>\n- ZealiD Qualified<sup>*</sup>\n<sup>*</sup> These verifications can not be used in any other position than the last.\n",
```

Comment:
I would suggest you keep all parameters descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).